api/server.go

```diff
@@ -284,9 +285,21 @@ func loggingStreamInterceptor(logger *zerolog.Logger) grpc.StreamServerIntercept
 	}
 }
 
+func RedactAuthToken(req interface{}) interface{} {
+	val := reflect.ValueOf(req).Elem()
+	for i := 0; i < val.NumField(); i++ {
+		field := val.Field(i)
+		if field.Kind() == reflect.String && (val.Type().Field(i).Name == "RegistryAuthToken") {
```

Comment:
This is a great idea, can you also add redactions to env vars? so if Name contains the phrase KEY or something we redact

Reply:
yessir